src/ProfilerEvent.php

```diff
@@ -80,7 +80,7 @@ public function setProfiler(Profiler $profiler)
      *
      * @return ReportInterface
      */
-    public function getReport()
+    public function getReport(): ReportInterface
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!